components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/OAuthUtil.java

```diff
@@ -210,9 +210,7 @@ public static void clearOAuthCache(String consumerKey, AuthenticatedUser authori
             userId = resolveUserIdFromUsername(authorizedUser);
             if (StringUtils.isEmpty(userId)) {
                 // Masking getLoggableUserId as it will return the username because the user id is not available.
-                LOG.error("User id cannot be found for user: " + (LoggerUtils.isLogMaskingEnable ?
-                        LoggerUtils.getMaskedContent(authorizedUser.getLoggableUserId()) :
-                        authorizedUser.getLoggableUserId()));
+                LOG.error("User id cannot be found for user: " + authorizedUser.getLoggableMaskedUserId());
```

Comment:
This returns authorizedUser.getLoggableMaskedUserId() the username in error scenarios.